libraries/chain/transaction.cpp

```diff
@@ -503,11 +503,12 @@ packed_transaction::packed_transaction(packed_transaction_v0&& other, bool legac
 
 packed_transaction_v0_ptr packed_transaction::to_packed_transaction_v0() const {
    const auto* sigs = get_signatures();
-   const auto* context_free_data = get_context_free_data();
-   signed_transaction strx( transaction( get_transaction() ),
+   auto& legacy = std::get<prunable_data_type::full_legacy>(prunable_data.prunable_data);
+
+   return std::make_shared<const packed_transaction_v0>( packed_trx,
                             sigs != nullptr ? *sigs : vector<signature_type>(),
-                            context_free_data != nullptr ? *context_free_data : vector<bytes>() );
-   return std::make_shared<const packed_transaction_v0>( std::move( strx ), get_compression() );
+                            legacy.packed_context_free_data,
+                            compression.value );
```

Comment:
The `.value` is not needed on `compression.value`

Reply:
Fixed.